test/onnx/test_onnx_export.py

```diff
@@ -19,27 +20,73 @@
 sys.path.append(pytorch_test_dir)
 
 
+def export_to_onnx(
+    model: Union[torch.nn.Module, torch.jit.ScriptFunction],
+    input: Tuple[torch.Tensor],
+    custom_ops: Optional[
+        Iterable[
+            Tuple[
+                str,
+                Union[contextlib.AbstractContextManager, contextlib.ContextDecorator],
+                int,
+            ]
+        ]
+    ] = None,
+    mocks: Optional[Iterable[unittest.mock.patch]] = None,
+    operator_export_type: OperatorExportTypes = OperatorExportTypes.ONNX,
+    opset_version=torch.onnx.symbolic_helper._export_onnx_opset_version,
```

Comment:
type for opset_version?

```suggestion
    opset_version: int = torch.onnx.symbolic_helper._export_onnx_opset_version,
```